### PR TITLE
Add editable uv scale for terrains

### PR DESCRIPTION
**Pull request by jpooleycodes**
Add UV scale slider in the terrain settings tab that persists and is stored in MetaTerrain

https://user-images.githubusercontent.com/28971753/164090477-356712c9-e8f1-4d20-8d40-4a3a688c53c1.mp4

### CODE REVIEW

File: commons/src/main/com/mbrlabs/mundus/commons/assets/meta/MetaLoader.java [hunk 1]
```diff
@@ -57,6 +57,7 @@ private void parseTerrain(Meta meta, JsonValue jsonTerrain) {
 
         final MetaTerrain terrain = new MetaTerrain();
         terrain.setSize(jsonTerrain.getInt(MetaTerrain.JSON_SIZE));
+        terrain.setUv(jsonTerrain.getFloat(MetaTerrain.JSON_UV_SCALE, 60));
```
Comment (Dgzt) [inline commons/src/main/com/mbrlabs/mundus/commons/assets/meta/MetaLoader.java:60]: I think the value 60 would be better in a constant file as a constant and you can use it in every place. What do you think?

Comment (jpooleycodes) [reply commons/src/main/com/mbrlabs/mundus/commons/assets/meta/MetaLoader.java:60]: hmm yeah, could use it there and in the uv vector initialization, trying to think if mundus already has a constants file

Comment (jpooleycodes) [reply commons/src/main/com/mbrlabs/mundus/commons/assets/meta/MetaLoader.java:60]: Could put is as a public static in Terrain class actually. It already has some other default values there

Comment (jpooleycodes) [reply commons/src/main/com/mbrlabs/mundus/commons/assets/meta/MetaLoader.java]: done

Comment (Dgzt) [reply commons/src/main/com/mbrlabs/mundus/commons/assets/meta/MetaLoader.java]: Nice :) Thanks

**Pull request merged**